core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -70,6 +70,7 @@ public class DoraCacheFileSystem extends DelegatingFileSystem {
   private final DoraCacheClient mDoraClient;
   private final FileSystemContext mFsContext;
   private final boolean mMetadataCacheEnabled;
+  private static final long DEFAULT_MOCK_BLOCK_SIZE = 33554432L;
```

Comment:
how you get this value?

Reply:
would it be helpful if you use the configured worker page size?

Reply:
You're right. I should get the value from the configuration.